wrappers/javascript/react-native/tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./build",
+    "skipLibCheck": true,
+    "types": ["react-native"]
+  },
+  "include": ["src/**/*"],
+  "exclude": ["example"]
```

Comment:
I think this is not needed anymore right as the example will be at the same level of the react-native / node packages right?